img2dataset/blurrer.py

```diff
@@ -0,0 +1,92 @@
+"""blurrer module to blur parts of the image"""
+
+import numpy as np
+
+import albumentations as A
+
+
+class BoundingBoxBlurrer:
```

Comment:
adding a simple unit test for this would be great
for example put in test folder one image and one expected blurred image, and then you run it and check output is expected output

Reply:
Got it - I'll add the unit test and reply again soon.